src/vectorchord/usage/search.md

```diff
@@ -23,6 +23,32 @@ For a given category, get the nearest 10 neighbors to a vector
 SELECT 1 FROM items WHERE category_id = 1 ORDER BY embedding <#> '[0.5,0.5,0.5]' limit 10
 ```
 
+## Range Filter
+
+To query vectors within a certain distance range, you can use the following syntax.
+```SQL
+-- Query vectors within a certain distance range
+SELECT * FROM items WHERE embedding <<->> sphere('[0.24, 0.24, 0.24]'::vector, 0.012);
+```
+
+The expression `sphere('[0.24, 0.24, 0.24]'::vector, 0.012)` refers to a spherical region with its center at `'[0.24, 0.24, 0.24]'` and a radius of `0.012`.
+
+The expression `embedding <<->> sphere('[0.24, 0.24, 0.24]'::vector, 0.012)` evaluates to `true` if the embedding is within the spherical region. Semantically, this is equivalent to `embedding <-> '[0.24, 0.24, 0.24]' < 0.012`.
+
+However, the expression `embedding <<->> sphere('[0.24, 0.24, 0.24]'::vector, 0.012)` can be correctly handled by the `vchordrq` index, while the expression `embedding <-> '[0.24, 0.24, 0.24]' < 0.012` can't.
+
+These comparison operators are used for range filter:
+
+| Operator | Description                           |
+| -------- | ------------------------------------- |
+| `<<->>`  | Tests if L2 distance <= threshold     |
+| `<<#>>`  | Tests if inner product <= threshold   |
+| `<<=>>`  | Tests if cosine distance <= threshold |
```

Comment:
Please don’t write this kind of "good enough to understand if readers think twice" text. Readers should not play word games in the document.